src/index.ts

```diff
@@ -92,6 +104,20 @@ const main = async (argv: string[]): Promise<any> => {
   }
   switch (args._[0]) {
     case "batch":
+      if ("ni" in args) {
+        return batch(
+          // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+          // @ts-ignore yargs typing is shit
+          args.rawDir,
+          args.batchedDir,
+          {
+            schemaPath: args.schema,
+            version: args.oav3 ? "open-attestation/3.0" : "open-attestation/2.0",
+            ni: "noninteractive"
```

Comment:
rename the attribute to nonInterective and only provide args.nonInteractive. typescript will be troublesome in that case I think we should consider casting arg to any for the moment